packages/next/src/server/app-render/create-error-handler.tsx

```diff
@@ -165,7 +162,10 @@ export function createHTMLErrorHandler(
       }
     } else {
       err.digest = stringHash(
-        err.message + (errorInfo?.stack || err.stack || '')
+        err.message +
+          (typeof (errorInfo as any)?.stack === 'string'
```

Comment:
we don't need this check as stack doesn't exist on errorInfo

Reply:
updated to use the componentStack if it exists